packages/react-provider/src/hooks/use-track-event.ts

```diff
@@ -8,8 +8,18 @@ const TRACK_EVENT = `
   }
 `;
 
+const BATCH_TRACK_EVENT = `
+  mutation BatchTrackEvent($eventType: String!, $messageIds: [String]!) {
+    batchTrackEvent(eventType: $eventType, messageIds: $messageIds) {
+      ids
+    }
+  }
+`;
+
 const useTrackEvent = () => {
-  return useMutation(TRACK_EVENT);
+  const [, trackEvent] = useMutation(TRACK_EVENT);
+  const [, batchTrackEvent] = useMutation(BATCH_TRACK_EVENT);
+  return [trackEvent, batchTrackEvent];
```

Comment:
i feel like we should just return an object here

Reply:
Agree